orc8r/cloud/go/obsidian/swagger/spec_servicer_test.go

```diff
@@ -15,43 +15,53 @@ package swagger_test
 
 import (
 	"context"
+	"errors"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"testing"
 
 	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/obsidian/swagger/mocks"
 	"magma/orc8r/cloud/go/obsidian/swagger/protos"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestSpecServicer_NewSpecServicerFromFile(t *testing.T) {
+	// missing default dir / spec files should not panic
+	assert.NotNil(t, swagger.NewSpecServicerFromFile("foo"))
```

Comment:
Nit: I prefer `err := ...` then `assert.NotNil(t, err)`, just for readability

Reply:
We end up with this shadowing problem either way considering the `val, err := ...` pattern, so I don't view that as a strong reason. My nit here is basically just from my other comment about the "readability" of the line, where I think it flows better to do `action \n assert` rather than `assert(action)`, if that makes sense